DOL.WHD.Section14c.Web/src/modules/services/authService.js

```diff
@@ -56,20 +56,23 @@ module.exports = function(ngModule) {
           stateService.user = data;
           if (data.organizations.length > 0) {
             stateService.ein = data.organizations[0].ein; //TODO: Add EIN selection?
+          } else {
+            d.resolve();
```

Comment:
Should this include a `return` statement?  Otherwise the function will continue to execute, maybe with side effects.  Once a promise is resolved, it should not resolve or reject again later, which is what will happen as-is.

Reply:
agreed and fixed.